src/test/java/uk/sky/cqlmigrate/CqlMigratorImplTest.java

```diff
@@ -217,10 +224,10 @@ public void shouldThrowExceptionForInvalidBootstrap() throws Exception {
         Collection<Path> cqlPaths = singletonList(cqlPath);
 
         //when
-        MIGRATOR.migrate(CASSANDRA_HOSTS, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
+        MIGRATOR.migrate(CASSANDRA_HOSTS, LOCAL_DC, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
 
         //then
-        cluster.connect(TEST_KEYSPACE);
+        //throws exception
```

Comment:
Could we catch the throwable and assert it's type instead of nothing? This has highlighted that it's not obvious which functional call the exception was previously being thrown from